operator/merkle_tree/lib/src/lib.rs

```diff
@@ -1,46 +1,77 @@
 use lambdaworks_crypto::merkle_tree::merkle::MerkleTree;
-use batcher::types::{VerificationBatch, VerificationData};
+use batcher::types::{VerificationCommitmentBatch, VerificationData};
+use hex;
+use serde_json;
+use std::fs::File;
```

Comment:
We can either remove this imports or remove the ones that are being imported in the tests module